searcher/analyse/analyse.go

```diff
@@ -0,0 +1,61 @@
+package analyse
+
+import (
+	"log"
+	"strings"
+
+	"github.com/sleepypioneer/resource-searcher/searcher/resource"
+)
+
+var (
+	keywords    = []string{"python", "webservice", "server", "Django", "list"}
+	punctuation = []string{".", ",", "!", ":", ";", "'", "/", "-", "_", "*", "&", "{", "}", "[", "]"}
+)
+
+func stringInSlice(a string, list []string) bool {
+	for _, b := range list {
+		if b == a {
+			return true
+		}
+	}
+	return false
+}
+
+// WordCount returns total number of words in the article
+func WordCount(content []string) int {
+	wordCount := 0
+	// for each paragraph in the article we count how many words it contains.
+	for _, p := range content {
+		// remove all punction in paragraph
+		for _, punc := range punctuation {
+			p = strings.ReplaceAll(p, punc, "")
```

Comment:
Good enough for the moment, but we will extend this in the future: Text normalisation is pretty wild and endless :sweat_smile: